lib/Makefile

```diff
@@ -262,9 +264,15 @@ install-static: libzstd.a
 install-shared: libzstd
 	@echo Installing shared library
 	@$(INSTALL) -d -m 755 $(DESTDIR)$(LIBDIR)/
+ifneq (,$(filter MINGW%,$(shell uname)))
+	@$(INSTALL) -d -m 755 $(DESTDIR)$(BINDIR)/
+	@$(INSTALL_PROGRAM) $(LIBZSTD) $(DESTDIR)$(BINDIR)
```

Comment:
Why does `$(LIBZSTD)` go in `$(BINDIR)`?

Reply:
That's a good point.
No idea what is the "proper" directory layout for `msys2`.
This part comes from @vtorri patch.